scripts/zones/Chamber_of_Oracles/mobs/Purson.lua

```diff
@@ -0,0 +1,27 @@
+-----------------------------------
+-- Area: Chamber of Oracles
+--  Mob: Purson
+-- KSNM30: The Scarlet King
+-----------------------------------
+local entity = {}
+
+entity.onMobSpawn = function(mob)
+    mob:setMod(xi.mod.DOUBLE_ATTACK, 40)
+end
+
+entity.onMobEngaged = function(mob)
+    mob:setLocalVar("timer", os.time() + 10)
+    mob:useMobAbility(800)
+end
+
+entity.onMobFight = function(mob, target)
+    if mob:getLocalVar("timer") < os.time() then
+        mob:useMobAbility(695)
```

Comment:
Add a TP check here to give back to the mob on finish of the ability like in Hostile Herbivores